### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by bparish628**
Can you think of anything else that needs to be changed? I moved out the installation to another file and created a new section for development tests (Development)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,12 +46,12 @@ Assumptions made about production:
 - Apache httpd server is used to serve app using mod_wsgi.
 - All static files are served from Apache using a redirect from url `/static/` to a static documents folder. This folder is created using `python manage.py collectstatic`.  This is not necessary in development.
 
-##License and Copyright
-All source code is covered by the MIT license.  This license is located in the LICENSE.txt file at the root of the project.
+## License and Copyright
+All source code is covered by the MIT license.  This license can be found [here](https://github.com/jakeharding/repo-health/blob/master/LICENSE.txt).
```
Comment (jakeharding) [inline README.md:51]: Should we put an actual copyright here?  Something like `© 2017 Jake Harding and Benji Parish`?

**Review by jakeharding — COMMENTED** [review]
I wonder if we should add an explicit copyright to the readme.

File: README.md (hunk 1 above)
Comment (bparish628) [reply README.md:51]: Yeah, lets do that.

**Review by jakeharding — APPROVED** [review]
Thanks.

**Pull request merged**